### PR TITLE
the rest of about me edit profile page, fix gender and pronoun fill-in

### PR DESCRIPTION
**Pull request by andreaowu**
Screen Shot 2021-03-01 at 11 25 48 AM
Screen Shot 2021-03-01 at 11 35 18 AM
Screen Shot 2021-03-01 at 11 35 26 AM

### CODE REVIEW

File: app/frontend/src/features/constants.ts [hunk 1]
```diff
@@ -42,16 +42,21 @@ export const SECTION_LABELS = {
 };
 
 // Edit Profile
+export const AGE = "Age";
+export const AGE_MAX = 100;
+export const AGE_MIN = 21;
```
Comment (lucaslcode) [inline app/frontend/src/features/constants.ts]: Unused?

File: app/frontend/src/features/constants.ts [hunk 2]
```diff
@@ -42,16 +42,21 @@ export const SECTION_LABELS = {
...
 export const COUNTRIES_VISITED = "Countries I've Visited";
 export const COUNTRIES_LIVED = "Countries I've Lived In";
 export const FEMALE = "Female";
 export const FEMALE_FORM_VALUE = "female";
+export const FEMALE_PRONOUNS = "she / her";
```
Comment (lucaslcode) [inline app/frontend/src/features/constants.ts]: These are man/woman on the signup page, I thought that's more gender-related.
In any case, I think the value should be capitalized, so it appears that way on the profile.

File: app/frontend/src/features/profile/edit/ProfileForm.tsx [hunk 3]
```diff
@@ -56,6 +62,12 @@ const useStyles = makeStyles((theme) => ({
     "& > .MuiInputBase-root": {
       width: "100%",
     },
+    "& .MuiOutlinedInput-notchedOutline": {
+      borderColor: theme.palette.grey[500],
+    },
+    "& .MuiOutlinedInput-root:hover .MuiOutlinedInput-notchedOutline": {
+      borderColor: theme.palette.grey[900],
+    },
```
Comment (lucaslcode) [inline app/frontend/src/features/profile/edit/ProfileForm.tsx]: Is this to match the hifi more closely? I'm not sure it's 100% necessary, but if we want this, I think it should be in mui theme overrides or in a component - unless these colours shouldn't apply on other pages

**Review by lucaslcode — COMMENTED** [review]
I'm sorry I didn't notice in the previous PRs, but the overview section is displaying blanks. I think maybe there it's okay to omit blank ones, since age, gender and "coucher since" will always be shown? Also, brackets for pronouns shouldn't appear if there are no pronouns. Feel free to leave those for another pr though if you think it's too unrelated.

Other stuff just minor, if you don't want to change anything, request again and I'll approve

Edit: And fluent languages are being displayed with "," in between instead of ", "

File: app/frontend/src/features/constants.ts (hunk 2 above)
Comment (andreaowu) [reply app/frontend/src/features/constants.ts]: I feel like woman / man is weird...I never see those as gender categories on any sites. Why did we pick woman / man over female / male?

File: app/frontend/src/features/profile/edit/ProfileForm.tsx (hunk 3 above)
Comment (andreaowu) [reply app/frontend/src/features/profile/edit/ProfileForm.tsx]: This is currently in [TextField](https://github.com/Couchers-org/couchers/blob/24e81c4757e6079907c27713a2c78167b667b3fd/app/frontend/src/components/TextField.tsx#L17-L22), but reversed. `TextField` is used a bunch throughout the UI so I didn't want to change it and mess things up

**Comment by andreaowu** [comment]
re: omitting blank ones - people's profiles aren't going to look uniform, which I don't really like. I think if people don't fill it in, we can just replace it with "Ask me" or just leave it blank. 

I'll fix the brackets and languages.

File: app/frontend/src/features/profile/edit/ProfileForm.tsx (hunk 3 above)
Comment (lucaslcode) [reply app/frontend/src/features/profile/edit/ProfileForm.tsx]: I think if this is the correct way, then change them all! :D

File: app/frontend/src/features/constants.ts (hunk 2 above)
Comment (lucaslcode) [reply app/frontend/src/features/constants.ts]: Because I am a fool

**Review by lucaslcode — DISMISSED** [review]
No need to squash commits, it can be good to see the process and makes reviewing a bit easier

File: app/frontend/src/features/constants.ts (hunk 2 above)
Comment (andreaowu) [reply app/frontend/src/features/constants.ts]: LOL no you're not!

**Review by lucaslcode — APPROVED** [review]

**Pull request merged**